google/provider.go

```diff
@@ -297,3 +298,39 @@ func convertStringArr(ifaceArr []interface{}) []string {
 	}
 	return arr
 }
+
+// flattenComputeMetadata transforms a list of MetadataItems (as returned via the GCP client) into a simple map from key
```

Comment:
I moved it to metadata.go as it works with both project metadata and instance metadata